templates/search.twig

```diff
@@ -12,6 +12,9 @@
 			<div class="without-search-result row clearfix">
 				<div class="col-md-12">
 					<h2 class="result-statement">{{ page_title|e('wp_kses_post')|raw }}</h2>
+					{% if query_time %}
```

Comment:
Mainly for checking the performance of these queries on develop sites, only shown for logged in users but we probably will remove this again.